src/de/jost_net/JVerein/io/AbrechnungSEPA.java

```diff
@@ -459,10 +459,13 @@ private JVereinZahler abrechnungMitgliederSub(AbrechnungSEPAParam param, Progres
         zahler.setPersonTyp(JVereinZahlerTyp.MITGLIED);
         zahler.setBetrag(
             BigDecimal.valueOf(betr).setScale(2, RoundingMode.HALF_UP));
-        new BIC(mZahler.getBic()); // Pr�fung des BIC
-        zahler.setBic(mZahler.getBic());
-        new IBAN(mZahler.getIban()); // Pr�fung der IBAN
+        IBAN i = new IBAN(mZahler.getIban()); // Pr�fung der IBAN
         zahler.setIban(mZahler.getIban());
+        //Wenn BIC nicht vorhanden versuchen sie automatisch zu ermitteln
+        if(mZahler.getBic() == null || mZahler.getBic().length() == 0)
+          zahler.setBic(i.getBIC());
+        else zahler.setBic(mZahler.getBic());
+        new BIC(zahler.getBic()); // Pr�fung des BIC
```

Comment:
Da ist noch ein Encoding-Problem mit den german umlauts in den comments.

Reply:
JVerin ist in ISO-8859-15 Codiert, gitHub verwendet UTF8 daher werden die Umlaute leider immer nicht richtig dargestellt.